bolt/include/bolt/Rewrite/DWARFRewriter.h

```diff
@@ -89,6 +89,13 @@ class DWARFRewriter {
   /// Store Rangelists writer for each DWO CU.
   RangeListsDWOWriers RangeListsWritersByCU;
 
+  using LegacyRangesDWOWriers =
+      std::unordered_map<uint64_t, std::unique_ptr<DebugRangesSectionWriter>>;
+  /// Store Ranges writer for each DWO CU.
+  LegacyRangesDWOWriers LegacyRangesWritersByCU;
```

Comment:
Maybe best to just have one of these unordered_map that maps dwoID to a struct containing std::unique_ptr<DebugRangesSectionWriter>> and DIE *.